tools/validators/instance_validator/validate/handler.py

```diff
@@ -27,6 +27,9 @@
 from validate import telemetry_validator
 from yamlformat.validator import presubmit_validate_types_lib as pvt
 
+# pylint: disable=consider-using-f-string
```

Comment:
it is better to fix those here, since we will have to fix them when importing internally

Reply:
i will try it this way, since it will be a bit of work to go adjust everything. if g3sync fails, then we will revise.

Reply:
g3 sync won't fail since it also uses pylint. They call it gpylint (sparkles)